rat/fit.py

```diff
@@ -139,18 +157,84 @@ def save(self, folder, overwrite=False):
             f.write(self.__class__.__name__)
 
 
+@dataclass
+class GeneratedQuantitiesFunctionGenerator(RatWalker):
+    variable_table: VariableTable
+    trace_table: TraceTable
+    parameters: Dict[str, Any]
+    traced_nodes: List[ast.Variable] = field(default_factory=list)
+
+    def process_node(self, node: ast.ModelBase):
```

Comment:
I'm not a big fan of having an entrypoint here that isn't `.walk()`. I'd rather this be walk_Expression or something.

Reply:
Not optimal, but I think it's fine if you make the notion of it being an entrypoint clear.

Reply:
My suspicion is that I'm trying to do too many things with one walker and I should try to separate these. I'm not sure, but all the other walkers are the same. I'd like to not change that pattern.